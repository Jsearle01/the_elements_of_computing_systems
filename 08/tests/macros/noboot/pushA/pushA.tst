load pushA.asm,
output-file pushA.out,
compare-to pushA.cmp,
output-list
    D%D1.6.1
    RAM[0]%D1.6.1
    RAM[256]%D1.6.1
    RAM[257]%D1.6.1
    RAM[258]%D1.6.1;

set A 2,
set RAM[0] 257,
set RAM[256] 1,

repeat 100 {
  ticktock;
}

output;

