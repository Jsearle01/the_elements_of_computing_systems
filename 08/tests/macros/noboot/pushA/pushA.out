|   D    | RAM[0] |RAM[256]|RAM[257]|RAM[258]|
|      2 |    258 |      1 |      2 |      0 |
